stress-testing/stress-test.py

```diff
@@ -0,0 +1,126 @@
+import asyncio
+import argparse
+import uuid
+import time
+
+from jupyterhub_client.api import JupyterHubAPI, JupyterAPI
+from jupyterhub_client.utils import parse_notebook_cells, tangle_cells
+
+async def run_user_notebook(hub_url, cells, token=None):
+    server_started = False
+    notebook_executed = False
+
+    hub = JupyterHubAPI(hub_url=hub_url, api_token=token)
+
+    try:
+        async with hub:
+            token = await hub.identify_token(hub.api_token)
+            service_name = token['name']
+            username = f'service-{service_name}-{uuid.uuid4()}'
+            user = await hub.ensure_user(username, create_user=True)
+            try:
+                if user['server'] is None:
```

Comment:
Can we use https://github.com/costrouc/jupyterhub_client/blob/7588e194942a7cf67418a6ec810e79c94004a510/jupyterhub_client/api.py#L62 instead?

Reply:
The reason why I didn't use `hub.ensure_server` is because I wanted to call `hub.ensure_user` myself (first thing that `ensure_server` does is to create the user) so that if the task cancellation happens after the user creation, I know I should delete the user as part of the cleanup phase. Do you think it makes sense?